### PR TITLE
:zap: Bump up the fhir-commons-utils and disable the auth pramaters

### PR DESCRIPTION
**Pull request by dubdabasoduba**
## Description of what I changed
- Updated the FHIR Commons utils dependencies 
- Disable the auth params for the `practitioner-details`

### CODE REVIEW

**Comment by rehammuzzamil** [comment]
Is this PR still needed to be reviewed/merged? @dubdabasoduba 

I see @ndegwamartin has added configs for the exemption here in this [PR](https://github.com/opensrp/fhir-access-proxy/pull/19).
We only need to keep a code for the removal of the authentication parameters, within the scope of this PR.

File: plugins/src/main/java/com/google/fhir/proxy/plugin/PermissionAccessChecker.java [hunk 1]
```diff
@@ -315,11 +314,13 @@ public Map<String, List<IQueryParameterType>> getMapForWhere(String keycloakUUID
...
       // Adding isAuthProvided
-      SpecialParam isAuthProvided = new SpecialParam();
+      // TODO this is no longer necessary after the update to remove the keycloak details from the
+      // response
+      /*      SpecialParam isAuthProvided = new SpecialParam();
       isAuthProvided.setValue("false");
       List<IQueryParameterType> l = new ArrayList<IQueryParameterType>();
       l.add(isAuthProvided);
-      hmOut.put(PractitionerDetails.SP_IS_AUTH_PROVIDED, l);
+      hmOut.put(PractitionerDetails.SP_IS_AUTH_PROVIDED, l);*/
```
Comment (ndegwamartin) [inline plugins/src/main/java/com/google/fhir/proxy/plugin/PermissionAccessChecker.java:323]: @dubdabasoduba should we remove these lines entirely?

File: resources/hapi_page_url_allowed_queries.json [hunk 2]
```diff
@@ -9,19 +9,19 @@
...
     {
-      "path": "Binary/ANY_VALUE",
+      "path": "Binary",
       "methodType": "GET",
       "queryParams": {
-
+        "_id":"ANY_VALUE"
       },
       "allowExtraParams": true,
       "allParamsRequired": false
```
Comment (ndegwamartin) [inline resources/hapi_page_url_allowed_queries.json:27]: We need to update the `allParamsRequired` to true to enforce that an `_id` param is required for the` /Binary` endpoint

Comment (dubdabasoduba) [reply resources/hapi_page_url_allowed_queries.json:27]: I see what you mean.

**Comment by dubdabasoduba** [comment]
> Is this PR still needed to be reviewed/merged? @dubdabasoduba
> 
> I see @ndegwamartin has added configs for the exemption here in this [PR](https://github.com/opensrp/fhir-access-proxy/pull/19). We only need to keep a code for the removal of the authentication parameters, within the scope of this PR.

@rehammuzzamil Yes, we already have the server deployed with this changes but we will merge it soon.

**Review by ndegwamartin — APPROVED** [review]

**Review by rehammuzzamil — APPROVED** [review]

**Pull request merged**